runtime/src/lib.rs

```diff
@@ -418,8 +418,11 @@ impl pallet_babe::Config for Runtime {
 	type EpochChangeTrigger = ExternalTrigger;
 	type EpochDuration = EpochDuration;
 	type ExpectedBlockTime = ExpectedBlockTime;
-	type HandleEquivocation =
-		BabeEquivocationHandler<Self::KeyOwnerIdentification, (), ReportLongevity>;
+	type HandleEquivocation = pallet_babe::EquivocationHandler<
+		Self::KeyOwnerIdentification,
+		pallet_octopus_lpos::FilterHistoricalOffences<OctopusLpos, Offences>,
```

Comment:
Don't call direcly, move this to import section,
https://github.com/debionetwork/debio-node/blob/56d2de71591f1d5c46cd5ffa103c84acf198580d/runtime/src/lib.rs#L55